### ISSUE TITLE
state.final() and query() deprecations warnings in typst (git hash 022f34c4)

### ISSUE THREAD

**Issue by swaits** [typst-community/glossarium#58]
**Description:**

It appears that `#state.final()` and `#query` no longer require `location` parameters.

**Diagnostics (deprecation warnings):**

```text
warning: call
ing `state.final` with a location is deprecated
   ┌─ @preview/glossarium:0.4.1/glossarium.typ:37:29
   │
37 │     let __glossary_entries = __glossary_entries.final(here())
   │                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   │
   = hint: try removing the location argument

warning: calling `query` with a location is deprecated
   ┌─ @preview/glossarium:0.4.1/glossarium.typ:20:4
   │
20 │ ╭     query(
21 │ │       selector(label(__glossary_label_prefix + key)).before(loc, inclusive: false),
22 │ │       loc,
23 │ │     )
   │ ╰─────^
   │
   = hint: try removing the location argument

warning: calling `query` with a location is deprecated
   ┌─ @preview/glossarium:0.4.1/glossarium.typ:25:4
   │
25 │     query(selector(label(__glossary_label_prefix + key)), loc)
   │     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   │
   = hint: try removing the location argument
```

**Latest (dev) typst installed via:**

```sh
cargo install --git https://github.com/typst/typst --locked typst-cli --force
```

**Comment by quachpas** [comment]
Thanks! Will take a look at the attached PR

**Comment by quachpas** [comment]
Solved in #52
